gnt/gnttab_stubs.c

```diff
@@ -73,26 +85,29 @@ stub_gnttab_unmap (value xgh, value array)
 {
         CAMLparam2 (xgh, array);
         int result;
-	xengnttab_handle* xgt = _G(xgh);
-	void* start_address = _M(array)->addr;
-	uint32_t count = _M(array)->len >> XEN_PAGE_SHIFT;
-	char s[64];
-
-	/* Check if this grant hasn't already been unmapped before */
-	if (start_address) {
-		caml_enter_blocking_section ();
-		result = xengnttab_unmap (xgt, start_address, count);
-		caml_leave_blocking_section ();
-		/* Avoid double unmapping by NULL-ing the pointer */
-		_M(array)->addr = NULL;
-		_M(array)->len = 0;
-
-		if (result != 0)
-		{
-			  int r = snprintf(s, 64, "Failed to unmap grant (errno %d, rc %d)", errno, result);
-			  caml_failwith(s);
-		}
-	}
+        xengnttab_handle *xgt = _G (xgh);
+        void *start_address = _M (array)->addr;
+        uint32_t count = _M (array)->len >> XEN_PAGE_SHIFT;
+        char s[64];
+
+        /* Check if this grant hasn't already been unmapped before */
+        if (start_address)
```

Comment:
I notice different formatting styles - has this been run through `make format`

Reply:
yes - in the first commit